### ISSUE TITLE
`pip install` of latest version does not reflect actual code in `main`

### ISSUE THREAD

**Issue by rwoodard-prog** [databrickslabs/dbx#621]
## Expected Behavior

Clean install of v0.8.7 followed by `dbx init` should reflect all code in `main`.  My test example is to look at the cookiecutter template in [setup.py](https://github.com/databrickslabs/dbx/blob/main/dbx/templates/projects/python_basic/render/%7B%7Bcookiecutter.project_name%7D%7D/setup.py#L28), where current code in repo shows:

```
    "dbx>=0.8"
```

## Current Behavior

Three issues (or my user error):

- after `dbx init`, `setup.py` contains `"dbx>=0.7,<0.8"`

- install issues `WARNING: dbx 0.8.7 does not provide the extra 'aws'`

- My git-fu is unable to find the commit shown in the install logs below, `5847742efa87e7104320467a3298ba22bd3c2e61`:

https://github.com/databrickslabs/dbx/commit/5847742efa87e7104320467a3298ba22bd3c2e61 returns 404.

## Steps to Reproduce (for bugs)

```
$ python --version
Python 3.9.5

$ which python
/opt/homebrew/Caskroom/miniconda/base/envs/db_rt_11.3_LTS_ML_sorta/bin/python

$ python -m pip install "dbx[aws] @ git+https://github.com/databrickslabs/dbx.git@v0.8.7"
Collecting dbx[aws]@ git+https://github.com/databrickslabs/dbx.git@v0.8.7
  Cloning https://github.com/databrickslabs/dbx.git (to revision v0.8.7) to /private/var/folders/jj/v8v5hy3d22ndmxj58b_736kw0000gp/T/pip-install-z9m0s_fs/dbx_2e0cfc6d17a5481b99b1a9670dc504df
  Running command git clone --filter=blob:none --quiet https://github.com/databrickslabs/dbx.git /private/var/folders/jj/v8v5hy3d22ndmxj58b_736kw0000gp/T/pip-install-z9m0s_fs/dbx_2e0cfc6d17a5481b99b1a9670dc504df
  Running command git checkout -q 5847742efa87e7104320467a3298ba22bd3c2e61
  Resolved https://github.com/databrickslabs/dbx.git to commit 5847742efa87e7104320467a3298ba22bd3c2e61
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
WARNING: dbx 0.8.7 does not provide the extra 'aws'
Requirement already satisfied: click<9.0.0,>=8.1.0 in /opt/homebrew/Caskroom/miniconda/base/envs/db_rt_11.3_LTS_ML_sorta/lib/python3.9/site-packages (from dbx[aws]@ git+https://github.com/databrickslabs/dbx.git@v0.8.7) (8.1.3)
...
Building wheels for collected packages: dbx
  Building wheel for dbx (pyproject.toml) ... done
  Created wheel for dbx: filename=dbx-0.8.7-py3-none-any.whl size=132736 sha256=ee1650fbd7ea5b2a1a1a2e6419a4fc7e855ed2c48af12d52c609ac3920b7d41e
  Stored in directory: /private/var/folders/jj/v8v5hy3d22ndmxj58b_736kw0000gp/T/pip-ephem-wheel-cache-ewg9mj7c/wheels/fd/5b/03/62cc83c0c821cb6302e8740186ae31b4b4d56339059de31ee3
Successfully built dbx
Installing collected packages: dbx
Successfully installed dbx-0.8.7

$ which dbx
/opt/homebrew/Caskroom/miniconda/base/envs/db_rt_11.3_LTS_ML_sorta/bin/dbx

$ dbx --version
[dbx][2022-11-30 11:14:25.803] 🧱 Databricks eXtensions aka dbx, version ~> 0.8.7

$ /opt/homebrew/Caskroom/miniconda/base/envs/db_rt_11.3_LTS_ML_sorta/bin/dbx init \
    -p "cicd_tool=GitHub Actions" \
    -p "cloud=AWS" \
    -p "project_name=test" \
    --no-input
[dbx][2022-11-30 11:10:35.721] Configuring new project from template python_basic ⚙
[dbx][2022-11-30 11:10:36.147] Configuring new environment with name default
[dbx][2022-11-30 11:10:36.149] Environment configuration successfully finished
[dbx][2022-11-30 11:10:37.353] Project configuration finished. You're all set to use dbx 🔥

$ grep dbx test/setup.py
    "dbx>=0.7,<0.8"
```

## Your Environment

See above.

**Comment by renardeinside** [comment]
hi @rwoodard-prog ,
this is available in 0.8.8 which is not yet released (most probably will be released after NY).

![Screenshot 2022-11-30 at 19 28 13](https://user-images.githubusercontent.com/4979594/204879041-a11626c4-8787-49d0-9a91-234abe83b588.png)

**Comment by rwoodard-prog** [comment]
OK, thanks for the quick reply, @renardeinside.
